packages/gatsby/src/pages/package.js

```diff
@@ -41,7 +41,12 @@ const PackagePage = ({ searchState, onSearchStateChange }) => {
   const [tags, setTags] = useState([]);
   const [owners, setOwners] = useState([]);
 
-  const packageName = typeof window !== 'undefined' && window.location.search.slice(1);
+  const [
+    /* leading slash */,
+    /* package/ */,
+    ...parts
+  ] = typeof window !== 'undefined' && window.location.pathname.split('/');
```

Comment:
Your slice likely comes from here, after the code has been transpiled. When running SSR, you'll be destructuring `undefined`.